### DISCUSSION TITLE
meson.build not found for subproject error while building the 1.7.0 tarball

### DISCUSSION THREAD

**Discussion by justvipul** [davatorium/rofi#1388] [(Possible) Issues]
the steps i performed,
```
$ wget https://github.com/davatorium/rofi/releases/download/1.7.0/rofi-1.7.0.tar.xz
$ tar -xf rofi-1.7.0.tar.xz
$ cd rofi-1.7.0
$ meson build setup
```
the error that comes is as follows,
[build-error](https://ezup.dev/p/szbspy)
and
[meson-log.txt](https://ezup.dev/p/zwdetl)

**Comment by DaveDavenport** [comment]
Thanks. that is indeed a bug. (I don't use meson myself, so it slipped through the cracks).

**Reply by DaveDavenport** [reply]
please use the autotools build system for now, I need to check this out with the maintainer of the submodules.

**Reply by DaveDavenport** [reply]
I have added a check to the build system to verify this. Hopefully this won't happen in the future.
